src/LabLog.Domain/Entities/School.cs

```diff
@@ -127,6 +140,12 @@ private void ApplyComputerAddedEvent(ILabEvent e)
             if (room.Computers == null) {throw new Exception("room.Computers is null");}
             Version = e.Version;
         }
+
+        private void ApplyStudentAssignedEvent(ILabEvent e)
+        {
+            var body = e.GetEventBody<StudentAssignedEvent>();
```

Comment:
You don't seem to be doing anything with the event data?

Reply:
Yeah, because all its for is essentially being displayed by the read model. There's no validation to occur, and it doesn't affect anything else's validation either. Later on I'll do something meaningful with it, like check that a student is not being assigned to two computers simultaneously, and might even implement a warning system, like studentx isn't allowed to use computers, or needs close supervision due to previous damage.

Reply:
so the same student can be assigned to many computers? Many students to the same computer?

Reply:
Yes to both of those. A student can't be assigned to two computers in a single submission, which I'll work on later, but because there is no method of unassigning students from computers (no one would ever do it!), it needs to be possible for a student to leave one computer and log into another, even though they are still assigned to the first. And definitely three students can be sharing a computer simultaneously.
Remember that this isn't business logic dictating what can and can't be, its just a recording mechanism of what is. Because there are so many things that could be done by the teacher in the room, the system needs to be very permissive to allow it to reflect a diverse range of actions. Its not the systems job to limit what the teacher can do, just to record it.